index.yaml

```diff
@@ -1,238 +1,246 @@
-  - name: StartETA
-
-- kind: Game
-  properties:
-  - name: NationAllocation
-  - name: StartETA
-
-- kind: Game
-  properties:
-  - name: Members.User.Id
-  - name: StartETA
-
-- kind: Game
-  properties:
-  - name: Started
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Closed
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Finished
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Variant
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Private
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: DisableConferenceChat
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: DisableGroupChat
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: DisablePrivateChat
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: NationAllocation
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Members.User.Id
-  - name: FinishedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Started
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Closed
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Finished
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Variant
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Private
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: DisableConferenceChat
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: DisableGroupChat
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: DisablePrivateChat
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: NationAllocation
-  - name: StartedAt
-    direction: desc
-
-- kind: Game
-  properties:
-  - name: Members.User.Id
-  - name: StartedAt
-    direction: desc
-
-
-# AUTOGENERATED
-
-# This index.yaml is automatically updated whenever the dev_appserver
-# detects that a new type of query is run.  If you want to manage the
-# index.yaml file manually, remove the above marker line (the line
-# saying "# AUTOGENERATED").  If you want to manage some indexes
-# manually, move them above the marker line.  The index.yaml file is
-# automatically uploaded to the admin console when you next deploy
-# your application using appcfg.py.
-
-
-- kind: GameResult
-  properties:
-  - name: Private
-  - name: Rated
-  - name: CreatedAt
-
-- kind: Glicko
-  properties:
-  - name: UserId
-  - name: CreatedAt
-    direction: desc
-
-- kind: Message
-  ancestor: yes
-  properties:
-  - name: CreatedAt
-
-- kind: Message
-  ancestor: yes
-  properties:
-  - name: CreatedAt
-    direction: desc
+    # RSS indexes
+
+    - kind: Game
```

Comment:
@tttppp So, I replaced these RSS indices for games (easiest to see if you look at the diff with `ignore whitespace`, e.g. https://github.com/zond/diplicity/pull/135/files?w=true) with composite indices (that re-use the sorting, but combine the non-sorted-on fields as necessary. I _think_ this works, but I hope there are tests for it?

Reply:
I doubt there are tests!  However I think I'm probably the only user of the feeds at the moment.  There were two purposes:
1. Provide a way for me to track games in particular variants
2. Provide a way to automatically report on specific games (e.g. during the Nexus tournament).

Reply:
OK, so we can just test if it works live :D